CONTRIBUTING.md

```diff
@@ -0,0 +1,3065 @@
+
+
+
```

Comment:
Do not use HTML in markdown files. There are example files you can simply copy and paste.